nntrainer/utils/node_exporter.h

```diff
@@ -56,11 +73,22 @@ class Exporter {
    * @param method method to export
    */
   template <typename... Ts>
-  void save_result(std::tuple<Ts...> &props, ExportMethods method) {
-    if (is_exported) {
-      throw std::invalid_argument("This exporter is already used");
+  void save_result(const std::tuple<Ts...> &props, ExportMethods method) {
+    switch (method) {
+    case ExportMethods::METHOD_STRINGVECTOR: {
+      auto callable = [this](auto &&prop, size_t index) {
```

Comment:
please add some brief about this lambda function here.
also, as for iterate_prop, callable is a template parameter, you can consider removing `index` for readability, and include it in other methods when needed.

Reply:
I'll put documentation here.